ProcessingShowPath/ProcessingShowPath.pde

```diff
@@ -9,19 +9,60 @@ float forwardTime;
 float backTime;
 int mistakes;
 int shortcuts;
+int startBiasX = -140;
+int startBiasZ = -50;
+int sizeX = 1200;
+int sizeY = 900;
+float stretchMultiplierX = 2.2;
+float stretchMultiplierY = 2.2;
+
+String[] filenames1;
+String[] filenames2;
+
+color forwardColor = color(0,0,255);
+color backColor = color(255,0,0);
+
+boolean drawForward = true;
+boolean drawBack = true;
 
 void setup() {
-  size(800,600);
-  PImage map = loadImage("minecraft.png");
+  size(1200,900);
+  PImage map = loadImage("isometric.png");
+  //map.filter(GRAY);
   background(map);
-  frameRate(1000000000);
-  reader = createReader("test.txt");
-  System.out.println("Started");
+  frameRate(30);
   loadPixels();  
-  c = color(0,0,255);
+  c = forwardColor;
+  println("Files for first condition: ");
+  filenames1 = listFileNames("C:/UnityProjects/P5Game/ProcessingShowPath/Condition1");
+  printArray(filenames1);
+
+  println("Files for second condition: ");
+  filenames2 = listFileNames("C:/UnityProjects/P5Game/ProcessingShowPath/Condition2");
+  printArray(filenames2);
+  
+  
+  System.out.println("Drawing first condition files");
+  for(int i = 0; i < filenames1.length; i++)
+  {
+    reader = createReader("Condition1/" + filenames1[i]);
+    drawPath();
+  }
+  
+  System.out.println("Drawing second condition files");
+  for(int i = 0; i < filenames2.length; i++)
+  {
+    reader = createReader("Condition2/" + filenames2[i]);
+    drawPath();
+  }
```

Comment:
I think these two loops could be added as one in the loop (to be done) when getting the files ☝️

Reply:
It will be easier to comment out this code if we just want 1 condition to be drawn.

Reply:
so we don't want all conditions to be drawn? Or are we going to copy-paste this 60 times later? :D

Reply:
maybe just have both ways of doing it? drawing one condition only and a for loop for all conditions?